README.rst

```diff
@@ -26,6 +26,30 @@ To run all benchmarks, do::
 
     $ venv/bin/cr8 run-track tracks/latest.toml [ -r result-host ]
 
+To run a single benchmark (spec)::
+
+    $ venv/bin/cr8 run-spec [--action setup|load_data|queries|teardown] specs/*TARGET_SPEC*.toml localhost:4200
```

Comment:
Not sure if it makes sense to point out `--action` and explain it in detail but not mention any of the other options.

How about keeping it at only `specs/specfile.toml localhost:4200`  and then refer to `run-spec --help` without further elaboration on `--action`? 

This kind of documentation duplication can get outdated quickly, in which case it confuses users more than that it helps.

If you think the `--help` output could be clarified you might want to make a PR against cr8 itself.